integration-test-common/src/test/java/org/apache/gravitino/integration/test/util/BaseIT.java

```diff
@@ -330,11 +344,18 @@ public void startIntegrationTest() throws Exception {
     serverConfig = new ServerConfig();
     customConfigs.put(ENTITY_RELATIONAL_JDBC_BACKEND_PATH.getKey(), file.getAbsolutePath());
     if (testMode != null && testMode.equals(ITUtils.EMBEDDED_TEST_MODE)) {
-      MiniGravitinoContext context = new MiniGravitinoContext(customConfigs, ignoreAuxRestService);
+      MiniGravitinoContext context =
+          new MiniGravitinoContext(
+              customConfigs, ignoreIcebergAuxRestService, ignoreLanceAuxRestService);
       miniGravitino = new MiniGravitino(context);
       miniGravitino.start();
       serverConfig = miniGravitino.getServerConfig();
     } else {
+      if (!ignoreLanceAuxRestService) {
```

Comment:
Why can't we move the code `ignoreLanceAuxRestService` and `ignoreRestAuxRestService` to L346 or before, I noticed that the same logic is there.